libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -173,13 +187,14 @@ std::optional<SortitionParamsChange> SortitionParamsManager::calculateChange(uin
   }
 
   const int32_t change = getChange(period, average_dag_efficiency);
-  config_.vrf += change;
+  VrfParams vrf = config_.vrf;
+  vrf += change;
```

Comment:
just making sure you wanted to change this from saving the change is class member into not saving it ?

Reply:
It moved here https://github.com/Taraxa-project/taraxa-node/pull/1306/files#diff-740ffa012af5a6c39db38b0cba5ee86455365dd3397a774838fd38b043e5a8c6R156. I noticed and thought that method with a name "calculateChange" shouldn't change anything. So I moved it and made this method const

Reply:
ah I see, ok